workflows/get_status.py

```diff
@@ -0,0 +1,178 @@
+import pathlib
```

Comment:
Can you rename this file to `get_species_status.py` or something similar?

Reply:
Sure